starfish/core/image/_filter/clip_value_to_zero.py

```diff
@@ -0,0 +1,105 @@
+from functools import partial
+from typing import Optional, Union
+
+import numpy as np
+import xarray as xr
+
+from starfish.core.imagestack.imagestack import ImageStack
+from starfish.core.util import click
+from ._base import FilterAlgorithmBase
+from .util import determine_axes_to_group_by
+
+
+class ClipValueToZero(FilterAlgorithmBase):
+    """
+    Image clipping filter that clips values below a minimum value and above a
+    maximum value. The filter then subtracts the minimum value from the
+    clipped image.
+
+    By default, the min and max values are set to 0.0 and None respectively,
+    which will result in the filter doing nothing.
+
+    This is a wrapper for :py:func:`numpy.clip`.
+
+    Parameters
+    ----------
+    v_min : float, optional
```

Comment:
Optional in the context of python typing has a different meaning than one might expect.  Consider removing this and just leaving the default.

Reply:
Thanks for bringing this to my attention!